### ISSUE TITLE
CT error: There already was a PhetioElement attached to this TreeNode with the same phetioID in the tree: 

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/greenhouse-effect#45]
```
greenhouse-effect : phet-io-studio-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/studio/?sim=greenhouse-effect&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22greenhouse-effect%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1623278478376%22%2C%22timestamp%22%3A1623280419503%7D
Uncaught Error: There already was a PhetioElement attached to this TreeNode with the same phetioID in the tree: greenhouseEffect.photonsScreen.model.positionProperty
Error: There already was a PhetioElement attached to this TreeNode with the same phetioID in the tree: greenhouseEffect.photonsScreen.model.positionProperty
at window.assert (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/phet-io-wrappers/common/js/assert.js:32:13)
at Function.insertPhetioElementIntoTree (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/studio/js/PhetioElement.js:86:17)
at Studio.onPhetioElementAdded (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/studio/js/studio.js:304:19)
at phetioElementAdded (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/studio/js/studio-main.js:42:10)
at Client.dispatch (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/phet-io-wrappers/common/js/Client.js:459:64)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/phet-io-wrappers/common/js/Client.js:328:26
at Array.forEach (<anonymous>)
at windowMessageListener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1623278478376/phet-io-wrappers/common/js/Client.js:310:29)
id: Bayes Chrome
Snapshot from 6/9/2021, 6:41:18 PM
```

Even with validation: false this is showing up, ill take a look.

**Comment by jessegreenberg** [comment]
To produce, you have to be in studio, go to the Photons screen and start sunlight.

**Comment by jessegreenberg** [comment]
I think this is because the ObservableArray managing photons is not instrumented, and has no tandem. When I give it a tandem, we need to add `toStateObject`, `fromStateObject` functions. I tried this and it didn't work out easily. I went to the documentation about instrumenting dynamically created elements at https://github.com/phetsims/phet-io/blob/master/doc/phet-io-instrumentation-technical-guide.md#dynamically-created-phet-io-elements, and it made me wonder if it is even necessary. So for now I decided to opt out of instrumenting photons. That gets the sim running and we can proceed with #43 for now, and come back to this later if we need.
